### PR TITLE
Portfolio finish

### PR DESCRIPTION
**Pull request by godybuoy**
Things done;
1. add styles to contact form by reducing border-radius.
2.  add type="submit" to the button of the contact form.
 I accidentally merged my pull request to https://github.com/godybuoy/Portfolio-setup-and-mobile-first-/pull/3

### CODE REVIEW

**Comment by joskalenda** [comment]
@godybuoy, did you read this comment [here](https://github.com/godybuoy/Portfolio-setup-and-mobile-first-/pull/3#issuecomment-1355418442)? to edit a pull request, please clink on these 3dots and select edit, but I can see you have created another again.

File: index.html [hunk 1]
```diff
@@ -198,12 +198,13 @@ <h1>contact me</h1>
...
+         love to help with it</p>
+         <form class="form-start"  action="https://formspree.io/f/xdovnkay"
+         method="POST">
           <input type="name" type="text" name="name" class="name" placeholder="write your name">
           <input type="email" type="email" name="email" class="email" placeholder="write your email">
-          <input type="message" type="text" name="message" class="message" placeholder="write your message">
-          <button class="start">get in touch</button>
+          <textarea type="message" type="text" name="message" class="message" placeholder="write your message"></textarea>
+          <button class="start" type="submit">Get in touch</button>
          </form>
```
Comment (joskalenda) [inline index.html:208]: - [ ] - [ OPTIONAL ] We have some optional suggestions here, it is about your input fields, it is said that all of them should have these attributes in the image below, I would strongly recommend you add them as it is contributing to your project security and best HTML practices.



- [ ] - [ OPTIONAL ]You have added a pull request summary but it is not descriptive at all, please next time take a look back to some of our 1st sessions on how to create a pull request summary.

- [ ] This review comment [here](https://github.com/godybuoy/Portfolio-setup-and-mobile-first-/pull/3#discussion_r1050079067) mentioned that your mobile version style is broken and you would have to fix it, and am having the same broken issue again, please consider making all the necessary changes before you resubmit your work again, please take a look at this record below

**Comment by godybuoy** [comment]
Hello , I have an open review kindly am not able to submit again

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @godybuoy , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by godybuoy** [comment]
Things done;
1. Removed the overflow to contact form.
2. Add some padding-top to text area.

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @godybuoy , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :+1: 
- [ ] [OPTIONAL]  Kindly add the hover effect to your button it will make your design stick to the Figma design as required in the project  :+1: 



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Comment by godybuoy** [comment]
finished portfolio mobile setup

**Pull request merged**